scripts/create_full_environment.sh

```diff
@@ -2,13 +2,16 @@
 
 set -o errexit
 
+export PATH=${PATH}:/usr/local/bin
+
+
 function error() {
     echo $@ 1>&2
 }
 
 # Check OS
 OS=$(awk -F= '/^ID=/ { print $2 }' /etc/os-release | tr -d '"')
-if [[ ! ${OS} =~ ^(centos)$ ]] && [[ ! ${OS} =~ ^(rhel)$ ]] && [[ ! ${OS} =~ ^(rocky)$ ]] && [[ ! ${OS} =~ ^(almalinux)$ ]]; then
+if [[ ! ${OS} =~ ^(ol)$ ]] && [[ ! ${OS} =~ ^(centos)$ ]] && [[ ! ${OS} =~ ^(rhel)$ ]] && [[ ! ${OS} =~ ^(rocky)$ ]] && [[ ! ${OS} =~ ^(almalinux)$ ]]; then
```

Comment:
what is this?

Reply:
From a commit that was prior to mine I guess, fixed it sorry